src/k-provider/ovp/services/playlist-service.js

```diff
@@ -0,0 +1,65 @@
+//@flow
+import OVPService from './ovp-service';
+import RequestBuilder from '../../../util/request-builder';
+
+const SERVICE_NAME: string = 'playlist';
+
+export default class OVPPlaylistService extends OVPService {
+  /**
+   * Creates an instance of RequestBuilder for playlist.getPlaybackContext
+   * @function getPlaybackContext
+   * @param {string} serviceUrl The service base URL
+   * @param {string} ks The ks
+   * @param {string} playlistId The playlist ID
+   * @returns {RequestBuilder} The request builder
+   * @static
+   */
+  static execute(serviceUrl: string, ks: string, playlistId: string): RequestBuilder {
+    const headers: Map<string, string> = new Map();
+    headers.set('Content-Type', 'application/json');
+    const request = new RequestBuilder(headers);
+    request.service = SERVICE_NAME;
+    request.action = 'execute';
+    request.method = 'POST';
+    request.url = request.getUrl(serviceUrl);
+    request.tag = `${SERVICE_NAME}-execute`;
+    request.params = {
+      ks: ks,
+      id: playlistId,
+      responseProfile: {
+        fields: 'id,name,description,thumbnailUrl,dataUrl,duration,msDuration,flavorParamsIds,mediaType,type,tags,dvrStatus',
```

Comment:
extract to `base-entry-response-profile.js` and import both in here and in base-entry-service